drift-codec-utils/src/main/java/com/facebook/drift/codec/utils/DataSizeToBytesThriftCodec.java

```diff
@@ -42,7 +42,7 @@ public DataSizeToBytesThriftCodec(ThriftCatalog thriftCatalog)
     }
 
     @CodecThriftType
-    public static ThriftType getThriftType()
+    public static ThriftType getThriftType(ThriftCatalog catalog)
```

Comment:
Please extract the change adding the `ThriftCatalog catalog` into a separate commit